schemas/2015-06-01/Microsoft.Cdn.json

```diff
@@ -0,0 +1,222 @@
+{
+    "id": "http://schema.management.azure.com/schemas/2015-06-01/Microsoft.Cdn.json#",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Microsoft.Cdn",
+    "description": "Microsoft Cdn Resource Types",
+    "resourceDefinitions": {
+        "profiles": {
+            "type": "object",
+            "description": "The CDN profile",
+            "properties": {
+                "type": {
+                    "enum": [ "Microsoft.Cdn/profiles" ]
+                },
+                "apiVersion": {
+                    "enum": [
+                        "2015-06-01"
+                    ]
+                },
+                "properties": {
+                    "type": "object",
+                    "properties": {
+                        "sku": {
+                            "type": "object",
+                            "properties": {
+                                "name": {
+                                    "someOf": [
```

Comment:
The error you see when using "oneOf" is actually a bug in our validation tool, we have fixed it last week. In the same fix, we also required "expression" support for all properties.
Please merge latest change from master branch and rerun.

For this one, let us use "oneOf". We should have exactly one match for a property value.